libsrc/psg/ay/arkos/msx_ram/arkos_c_stubs_msx_ram.asm

```diff
@@ -0,0 +1,66 @@
+;;
```

Comment:
Mmmm the real players are already at that location. Do you really think the C stubs should go in there as well? Do we have a directory for target-specific C stubs?

BTW, I have split all stubs files into one function per file in `libsrc/psg/ay/arkos/`.

Reply:
Classic just tries to keep all the target specific code in target/ - C stubs and asm files.

There's a couple of other ways to handle the multiple implementation selection:

- Create a new `-lmsxrom` library which is added to link line automatically when -subtype=rom is given
- Do some aliasing (eg `defc func = func_impl`) in the crt

Ideally we'd do the latter, however z80asm will always include `func_impl` in the final binary which isn't what we want.

I was originally thinking of the `msxrom` route until I saw your `#define`s which make it easy to tell at linking stage which routine got pulled in.

Reply:
> Classic just tries to keep all the target specific code in target/ - C stubs and asm files.

OK, so this seems very reasonable, I had this thought just after replying to you. I'll do it as you say.

My feeling was to separate stubs from the operative code, but indeed since all the Arkos code is in its own directory, there's no real problem in having all files in a single dir: the number of files is small and it's easy to know which ones are code and which ones are stubs.